doc/changes.rst

```diff
@@ -5,6 +5,7 @@ desitarget Change Log
 2.3.1 (unreleased)
 ------------------
 
+* Restore ``select_mock_targets`` to a working state [`PR #788`].
```

Comment:
You're missing the trailing underscore on 
```
[`PR #788`]
[`PR #788`_]
```
that would link this to the url for the PR.

Reply:
Done in https://github.com/desihub/desitarget/pull/788/commits/83c8a2421dcb6f91550b717804e57c7985cfcf0d.